app-emulation/spice-protocol/spice_protocol-0.14.4.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="SPICE protocol definitions"
+DESCRIPTION="Headers defining the SPICE-protocol"
+HOMEPAGE="https://www.spice-space.org/"
+COPYRIGHT="2009 Red Hat, Inc"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://www.spice-space.org/download/releases/spice-protocol-$portVersion.tar.xz"
+CHECKSUM_SHA256="04ffba610d9fd441cfc47dfaa135d70096e60b1046d2119d8db2f8ea0d17d912"
+SOURCE_DIR="spice-protocol-$portVersion"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	spice_protocol = $portVersion
+	devel:spice_protocol = $portVersion
+"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:meson
+	cmd:ninja
+	cmd:pkg_config$secondaryArchSuffix
```

Comment:
As this is any architecture, no need for $secondaryArchSuffix:
```diff
-	cmd:pkg_config$secondaryArchSuffix
+	cmd:pkg_config
```

Reply:
I am a little confused about secondaryArchSuffix, it's used for spicify the required platform suffix?

Reply:
Actually pkg-config isn't needed here so you can remove the line.
On the secondaryArchSuffix it will try to use pkg-config-x86 instead of pkg-config, it is only required if a package doesn't build or isn't available on the primary architecture, as this is any (primary) architecture there is no need for anything specific for secondary architecture (on 32bit)

Reply:
Thanks @Begasus , I removed pkg-config requires